css/styles.css

```diff
@@ -1,137 +1,461 @@
+  width: 20px;
+  height: 20px;
+}
+
+.m-icon {
+  background-image: url("../images/m-icon.png");
+  width: 20px;
+  height: 20px;
+}
+
+.cat-icon {
+  background-image: url("../images/cat-icon.png");
+  width: 20px;
+  height: 20px;
+}
+
+.fingers-icon {
+  background-image: url("../images/fingers-icon.png");
+  width: 20px;
+  height: 20px;
+}
+
+.projects {
+  padding: 30px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(327px, 1fr));
+  justify-content: space-around;
+  align-items: start;
+}
+
+.project {
+  margin: 20px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  justify-content: space-around;
+}
+
+#project1 {
+  order: 0;
+}
+
+#project2 {
+  order: 1;
+}
+
+#project3 {
+  order: 2;
+}
+
+#project4 {
+  order: 3;
+}
+
+#project1 .pimage {
+  margin-bottom: 5px;
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p1image.png");
+}
+
+#project2 .pimage {
+  margin-bottom: 5px;
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p2image.png");
+}
+
+#project3 .pimage {
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p3image.png");
+}
+
+#project4 .pimage {
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p4image.png");
+}
+
+.p-title {
+  width: 220px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  align-items: center;
+  text-align: left;
+  align-self: flex-start;
+}
+
+.p-title h2 {
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.details-list {
+  display: flex;
+  width: 295px;
+  height: 36px;
+  align-items: center;
+}
+
+.client {
+  width: 55px;
+  height: 16px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.spacedot {
+  width: 8px;
+  height: 8px;
+  background-image: url("../images/spacedot.png");
+}
+
+.role {
+  width: 90px;
+  height: 16px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.year {
+  width: 29px;
+  height: 16px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.p-description {
+  width: 295px;
+  height: 76px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages-list {
+  display: flex;
+  width: 295px;
+  height: 24px;
+  align-items: center;
+}
+
+.lang {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: auto;
+  height: 24px;
+  background: ##ebebff;
```

Comment:
- [ ] I noticed that there is a typo you added extra (#), kindly fix it 👍

Reply:
Done! thank you!